LICENSE

```diff
@@ -1,6 +1,8 @@
 BSD 3-Clause License
 
-Copyright (c) 2018, 
+Copyright (c) 2023 Mark Mc Mahon and Contributors
+https://github.com/pywinauto/injected/graphs/contributors
+https://pywinauto.readthedocs.io/en/latest/credits.html
```

Comment:
The only link to `injected` contributors makes sense. Because Mark McMahon most probably even doesn't know about `injected`. I'd suggest `Copyright (c) 2025 Vasily Ryabov, Ilya Naumov, Boris Galochkin, Alexander Makarov` at 2 lines above.